cmd/kubelet/app/options/options.go

```diff
@@ -114,7 +114,7 @@ func NewKubeletServer() *KubeletServer {
 			RktStage1Image:                 "",
 			RootDirectory:                  defaultRootDir,
 			SerializeImagePulls:            true,
-			StreamingConnectionIdleTimeout: unversioned.Duration{5 * time.Minute},
+			StreamingConnectionIdleTimeout: unversioned.Duration{0},
```

Comment:
@liggitt yes, that's the field. The other PR for this is #18552. I like your suggestion. Something more like 1 or 4 hours, perhaps.